weldx/asdf/schemas/weldx.bam.de/weldx/core/file-1.0.0.yaml

```diff
@@ -0,0 +1,84 @@
+%YAML 1.1
+---
+$schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
+id: "http://weldx.bam.de/weldx/core/file-1.0.0"
+tag: "tag:weldx.bam.de:weldx/core/file-1.0.0"
+
+title: |
+  Schema for a file.
+description: |
+  This schema describes a file by compiling all its meta data. Optionally, the whole file can also be stored in the
+  binary block of an asdf file.
+
+examples:
+  -
+    - A simple DHU-Groove
```

Comment:
Doesn't look like DHU-Groove to me 😉

Reply:
Then keep looking closer :D --- After several asdf test failures and copy-pasting it started looking like one to me ;) --- I'll fix it